src/tx-format/transaction.md

```diff
@@ -131,15 +131,18 @@ Creates a contract with contract ID as computed [here](../identifiers/contract-i
 ## TransactionMint
 
 The transaction is created by the block producer and is not signed. Since it is not usable outside of block creation or execution, all fields must be fully set upon creation without any zeroing.
+This means that the transaction ID must also include the correct `txPointer` value, not zeroed out.
 
-| name           | type                         | description                                          |
-|----------------|------------------------------|------------------------------------------------------|
-| `txPointer`    | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block. |
-| `outputsCount` | `uint8`                      | Number of outputs.                                   |
-| `outputs`      | [Output](./output.md)`[]`    | List of outputs.                                     |
+| name             | type                         | description                                          |
+|------------------|------------------------------|------------------------------------------------------|
+| `txPointer`      | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block. |
+| `inputContract`  | [Input](./input.md)          | The contract utxo that assets are minted to.         |
+| `outputContract` | [Output](./output.md)        | The contract utxo that assets are being minted to.   |
+| `mintAmount`     | `uint64`                     | The amount of funds minted.                          |
+| `mintAssetId`    | `byte[32]`                   | The asset IDs corresponding to the minted amount.    |
```

Comment:
Maybe it is better to keep the receipt root and produce burn and mint receipts for the indexer(as you mentioned in Slack). Otherwise, it requires additional handling from other parts of our stack.

Reply:
Let's keep it simple for now